src/JsonmlToReact.js

```diff
@@ -58,6 +59,11 @@ export default class JsonmlToReact {
     // reassign key in case `converter` removed it
     props.key = props.key || index;
 
+    // If it's a void element, don't create children
+    if (voidElementTags[type]) {
```

Comment:
Can we move this before https://github.com/CondeNast/jsonmltoreact/blob/9c41d76a62f00eea857c56e09207778407ee9b49/src/JsonmlToReact.js#L51 gets called?